flow-typed/forms.js

```diff
@@ -2,6 +2,7 @@
 
 import { type Address as GeoscapeAddress } from './geoscape/address-details'
 import { type Address as PointAddress } from './point/address-details'
+import { CivicaStreetName } from './civica/street-name'
```

Comment:
Need `type` don't we? Don't think this is valid flow

Reply:
Agreed.